class.jetpack.php

```diff
@@ -6547,7 +6673,7 @@ function concat_remove_style_loader_tag( $tag, $handle ) {
 		if ( in_array( $handle, $this->concatenated_style_handles ) ) {
 			$tag = '';
 			if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
-				$tag = "<!-- `" . esc_html( $handle ) . "` is included in the concatenated jetpack.css -->\r\n";
+				$tag = '<!-- `' . esc_html( $handle ) . "` is included in the concatenated jetpack.css -->\r\n";
```

Comment:
Not sure why one gets updated but not the other. Because of the carriage return?

Reply:
Correct, need `""` for the carriage returns to be processed as such and not literally as the string `\r\n`.